### DISCUSSION TITLE
with the tracking table, would the database size double (at the least)?

### DISCUSSION THREAD

**Discussion by koo9** [Mimetis/Dotmim.Sync#651] [Q&A]
any thoughts? 

also with the CRUD triggers, there are impact to the db performance as well.

**Comment by Mimetis** [comment]
> with the tracking table, would the database size double (at the least)?

No

> also with the CRUD triggers, there are impact to the db performance as well.

Yes

**Comment by koo9** [comment]
how much of the data is stored in the tracking table then?

**Reply by Mimetis** [reply]
the minimum

**Comment by vunder** [comment]
My experience in DMS says that database size increase related to indices in case you're using SQLite with GUID as primary key
